.gitlab-ci.yml

```diff
@@ -165,3 +165,50 @@ Release GCC Module:
     CMAKE_BUILD_TYPE: "Release"
     MODULE_BUILD: "yes"
   extends: .build
+
+Debug GPU Ubuntu:
+  retry:
+    max: 2
+  variables:
+    CC: "gcc"
+    CXX: "g++"
+    CCACHE_DIR: "${CI_PROJECT_DIR}/ccache"
+    CXXFLAGS: "-Wall -Werror"
+  stage: build
+  artifacts:
+    paths:
+      - ${CI_PROJECT_DIR}/votca
+    when: always
+  dependencies: []
+  cache:
+    paths:
+      - ccache/
+  before_script:
+    - source /opt/gromacs-2019/bin/GMXRC.bash
+  script:
+    - ccache -z
```

Comment:
It is a little unfortunate that we now have all this code duplication.

Reply:
I agree. I can play a bit to see if I can accommodate some of the duplicates in a single configuration